docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.8"
+services:
+    # basic drupal site installation with dependencies and permissions configured
+    # use `docker exec -ti journal-cms_app_1 /bin/bash` to get a shell
+    app:
+        build: .
```

Comment:
would be worth to put `./mysql` in a `.dockerignore` file, as it's accidentally picked up during a new `docker-compose buil` and sent to the daemon (this happen regardless of whether it's mentioned in `COPY~ statements)

Reply:
it's permissions are wonky as well after creation which makes the recreation a bit of a pain.

I imagine a local mysql container is only there for non-prod.